runtime/tr.source/trj9/il/symbol/J9MethodSymbol.cpp

```diff
@@ -538,6 +538,9 @@ static TR::RecognizedMethod canSkipZeroInitializationOnNewarrays[] =
    //TR::java_util_Arrays_copyOf,
    TR::java_io_Writer_write_lStringII,
    TR::java_io_Writer_write_I,
+   TR::java_lang_StringCoding_encode8859_1,
+   TR::java_lang_StringCoding_encodeASCII,
+   TR::java_lang_StringCoding_encodeUTF8,
```

Comment:
We should also be able to skip bound checks on the `getChar` recognized method. Just making sure: is this handled by the code in UnsafeFastPath?

Reply:
yes - getChar will be turned directly into a read - no bounds checking at all.